lib/util.js

```diff
@@ -78,4 +63,63 @@ util.findToken = function (req) {
   return token
 }
 
+function sizeof (obj) {
```

Comment:
Instead of having a separate function, can isSerializable be enhanced to return sizeOf. Right now 2 recusions are needed. then validation can be done in one.